deploy/README.md

```diff
@@ -0,0 +1,61 @@
+# Deploy on Google Cloud Platform
+
+### Setup
+
+
+* [Follow these instructions](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery/deploy)
+to bring up Elasticsearch in GKE and index dataset. If you are indexing
+sensitive data, set up Elasticsearch only, deploy the app and ensure IAP is
+properly enabled here before indexing your Elasticsearch instance.
+
+* If you haven't done so already, set the default project:
+
+  `gcloud config set project PROJECT_ID`
+
+* Create the App Engine application:
+
+  `gcloud app create`
+
+* Set up routing rules
+
+  `gcloud app deploy dispatch.yaml`
```

Comment:
I get:
```
~/data-explorer/deploy (ah/deploy): gcloud app deploy dispatch.yaml
Configurations to update:

descriptor:      [/usr/local/google/home/melchang/data-explorer/deploy/dispatch.yaml]
type:            [routing rules]
target project:  [google.com:api-project-360728701457]


Do you want to continue (Y/n)?  y

Updating config [dispatch]...failed.                                           
ERROR: (gcloud.app.deploy) Server responded with code [400]:
  Bad Request Unexpected HTTP status 400.
  Validation error: Invalid dispatch configuration - module 'api' does not exist. Upload a version of this module and try again.
```

Do you mind deleting all your deployments and trying again from scratch, to make sure these instructions work?

Reply:
Works for me with the updates instructions.